android_jni/avifandroidjni/src/androidTest/java/org/aomedia/avif/android/AvifDecoderTest.java

```diff
@@ -159,6 +159,25 @@ public void testDecodeUtilityClass() throws IOException {
     assertThat(AvifDecoder.decode(buffer, buffer.remaining(), bitmap)).isTrue();
   }
 
+  @Test
+  public void testDecodeScale() throws IOException {
+    if (image.isAnimated) {
+      return;
+    }
+    ByteBuffer buffer = image.getBuffer();
+    assertThat(buffer).isNotNull();
+    assertThat(AvifDecoder.isAvifImage(buffer)).isTrue();
+    Info info = new Info();
+    assertThat(AvifDecoder.getInfo(buffer, buffer.remaining(), info)).isTrue();
+    assertThat(info.width).isEqualTo(image.width);
+    assertThat(info.height).isEqualTo(image.height);
+    assertThat(info.depth).isEqualTo(image.depth);
+    assertThat(info.alphaPresent).isEqualTo(image.alphaPresent);
+    Bitmap bitmap = Bitmap.createBitmap(info.width/2, info.height/2, config);
```

Comment:
could you please add more cases?
* larger bitmap,
* bitmap with different aspect ratios than the source image

Reply:
yes